.github/workflows/scripts/scanner-fetch-nvd.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -euo pipefail
+
+base_url="https://services.nvd.nist.gov/rest/json/cves/2.0"
+dir="nvd-data"
+mkdir -p "$dir"
+
+info() { echo >&2 "$(date): INFO: $@"; }
+
+# Compression and upload
+compress_upload() {
+  tar -czf "$dir.tar.gz" -C "$dir" .
+  info "Compressed file size: $(stat -c %s "$dir.tar.gz") bytes"
+  mkdir -p "nvd-bundle" && mv "$dir.tar.gz" "nvd-bundle/$dir.tar.gz"
+  gsutil cp -r "nvd-bundle" "gs://scanner-v4-test/" || { info "Upload failed"; exit 1; }
+}
+
+dates() {
+  year=$1
```

Comment:
This was decided in the ADR https://github.com/stackrox/stackrox/blob/master/scanner/decisions/0005-scannerv4-read-cvss-data.md

Since we have the Api key for less strict rate limiting and we will eventually merge small files to yearly file, I don't think it will make much difference between 120 days and quarterly fetching. @jvdm

Reply:
Having the possibility to configure this doesn't invalidate the decision in the ADR because you can still break it into quarterly chunks if you want to.

It's OK to have the ability to change the yearly chunk size by using `date()`. It easily allows us to reduce or increase the chunk size if we eventually need to change the NVD API call pattern. Once the API key is available, it is also easy to go to larger chunks. But if we simply plan to go yearly with an API key, I would say don't even bother chunking the year and simply hard-code the end and start dates to Jan/01 and Dec/31.